docs/index.md

```diff
@@ -286,7 +286,7 @@ Core utilities such as Tracing, Logging, and Metrics will be available across al
 | Environment variable                         | Description                                                                            | Utility                   | Default               |
 |----------------------------------------------|----------------------------------------------------------------------------------------|---------------------------|-----------------------|
 | **POWERTOOLS_SERVICE_NAME**                  | Sets service name used for tracing namespace, metrics dimension and structured logging | All                       | `"service_undefined"` |
-| **POWERTOOLS_METRICS_NAMESPACE**             | Sets namespace used for metrics                                                        | [Metrics](./core/metrics) | `None`                |
+| **POWERTOOLS_METRICS_NAMESPACE**             | Sets namespace used for metrics                                                        | [Metrics](./core/metrics) | `"default_namespace"`                |
```

Comment:
Please remove extra `"` quotes from `default_namespace`

Reply:
Shall I remove the quotes from `"service_undefined"` above a well?

Reply:
Yes please, I missed it